packages/url_launcher/url_launcher_linux/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.3
+
+* Fix canLaunch detection for `file` scheme
```

Comment:
Nit: Fixes

(Per the repo's new CHANGELOG style guide)

Reply:
There was colliding commit having version ## NEXT by @stuartmorgan
I kept it as it was, but added this change to as new line.